### PR TITLE
Add Cheng-Zhen as a triager

### PR DESCRIPTION
**Pull request by dmathieu**
This proposes adding @scorpionknifes as a triager for the Go SIG.
While Cheng-Zhen's contributions are all in the contrib repository, we don't currently split roles between the two repos.

Cheng-Zhen's contributions in the log bridges, and his interest in being more involved with the SIG make him a nice fit to be a triager.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -629,6 +629,10 @@ should be canceled.
 
 ## Approvers and Maintainers
 
+### Triagers
+
+- [Cheng-Zhen Yang](https://github.com/scorpionknifes), mx51
```
Comment (scorpionknifes) [inline CONTRIBUTING.md:634]:
```suggestion
- [Cheng-Zhen Yang](https://github.com/scorpionknifes)
```
I'm currently not company-affiliated and won't be anytime soon (also switching roles next month).

Is it possible to keep company empty? thanks.

**Review by scorpionknifes — COMMENTED** [review]
Thanks @dmathieu!

File: CONTRIBUTING.md (hunk 1 above)
Comment (dmathieu) [reply CONTRIBUTING.md:634]: If you're working as an independant, how about marking it as such?

```suggestion
- [Cheng-Zhen Yang](https://github.com/scorpionknifes), Independant
```

Comment (scorpionknifes) [reply CONTRIBUTING.md:634]: Thanks, that works

**Review by scorpionknifes — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

**Review by dashpole — APPROVED** [review]

**Review by pellared — APPROVED** [review]
👍

**Pull request merged**